packages/valory/skills/abstract_round_abci/models.py

```diff
@@ -207,6 +210,23 @@ def synchronized_data(self) -> BaseSynchronizedData:
         """Get the latest synchronized_data if available."""
         return self.round_sequence.latest_synchronized_data
 
+    def get_acn_result(self) -> Any:
+        """Get the majority of the ACN deliverables."""
+        if len(self.address_to_acn_deliverable) == 0:
+            return None
+
+        threshold = consensus_threshold(self.synchronized_data.nb_participants)
```

Comment:
this here means (2/3) + 1 + 1. Since the agent doing the call is not sending a payload. In the future when we might want to consider the case when more than 1/3 of participants are down this will need adjustment.

Reply:
Yes suggest fixing this now. Just add: -1

Reply:
435afd87e35c3a99ed13c426e2d9e99ebe651e8f